core/src/test/java/com/datastax/oss/driver/internal/core/metadata/schema/parsing/AggregateParserTest.java

```diff
@@ -76,12 +77,13 @@ public void should_parse_modern_table() {
         .containsExactly(DataTypes.INT, DataTypes.INT);
     assertThat(aggregate.getStateType()).isEqualTo(DataTypes.INT);
 
-    FunctionSignature finalFuncSignature = aggregate.getFinalFuncSignature();
-    assertThat(finalFuncSignature.getName().asInternal()).isEqualTo("to_string");
-    assertThat(finalFuncSignature.getParameterTypes()).containsExactly(DataTypes.INT);
+    Optional<FunctionSignature> finalFuncSignature = aggregate.getFinalFuncSignature();
+    assertThat(finalFuncSignature).isPresent();
+    assertThat(finalFuncSignature.get().getName().asInternal()).isEqualTo("to_string");
+    assertThat(finalFuncSignature.get().getParameterTypes()).containsExactly(DataTypes.INT);
```

Comment:
You can rewrite this in a slightly prettier way with `assertThat(finalFuncSignature).hasValueSatisfying(signature -> { ... })`.